cpp/src/arrow/flight/sql/server_test.cc

```diff
@@ -21,6 +21,7 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include <arrow/flight/sql/example/sqlite_type_info.h>
```

Comment:
nit: Arrow headers are generally included with "" (also, please be consistent with the existing code)

Reply:
Fixed